.gitignore

```diff
@@ -54,3 +54,4 @@ fabric.properties
 Habitica/res/values/secret_strings.xml
 habitica.properties
 habitica.resources
+Habitica/google-services.json
```

Comment:
Where is this file generated? Can an example/template file be provided for travis?